bin/node-template/runtime/src/lib.rs

```diff
@@ -514,18 +514,20 @@ impl_runtime_apis! {
 			impl frame_system_benchmarking::Config for Runtime {}
 			impl baseline::Config for Runtime {}
 
-			let whitelist: Vec<TrackedStorageKey> = vec![
-				// Block Number
-				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef702a5c1b19ab7a04f536c519aca4983ac").to_vec().into(),
-				// Total Issuance
-				hex_literal::hex!("c2261276cc9d1f8598ea4b6a74b15c2f57c875e4cff74148e4628f264b974c80").to_vec().into(),
-				// Execution Phase
-				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef7ff553b5a9862a516939d82b3d3d8661a").to_vec().into(),
-				// Event Count
-				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef70a98fdbe9ce6c55837576c60c7af3850").to_vec().into(),
-				// System Events
-				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef780d41e5e16056765bc8461851072c9d7").to_vec().into(),
-			];
+			use crate::sp_api_hidden_includes_construct_runtime::hidden_include::traits::WhitelistedStorageKeys;
+			let whitelist: Vec<TrackedStorageKey> = AllPalletsWithSystem::whitelisted_storage_keys();
+			// let whitelist: Vec<TrackedStorageKey> = vec![
+			// 	// Block Number
+			// 	hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef702a5c1b19ab7a04f536c519aca4983ac").to_vec().into(),
+			// 	// Total Issuance
+			// 	hex_literal::hex!("c2261276cc9d1f8598ea4b6a74b15c2f57c875e4cff74148e4628f264b974c80").to_vec().into(),
```

Comment:
lets clean up comment code. were all these whitelisted?

Reply:
Yeah these were all whitelisted I wanted to figure out some way to verify that we're still generating roughly the same list, however printing is disabled here so I was wondering if you know a good way to introspect on `whitelist` here and see what is in it?

Reply:
you can write a test at the bottom of this file for sanity check.

Like so:

```rust
	#[test]
	fn check_whitelist() {
		use frame_support::traits::WhitelistedStorageKeys;
		use frame_support::traits::TrackedStorageKey;
		use sp_core::hexdisplay::HexDisplay;

		let whitelist: Vec<TrackedStorageKey> = AllPalletsWithSystem::whitelisted_storage_keys();
		for item in whitelist {
			println!("{:?}", HexDisplay::from(&item.key));
		}
		assert!(false);
	}
	```
	
Output was:

```
---- tests::check_whitelist stdout ----
26aa394eea5630e07c48ae0c9558cef702a5c1b19ab7a04f536c519aca4983ac
26aa394eea5630e07c48ae0c9558cef70a98fdbe9ce6c55837576c60c7af3850
26aa394eea5630e07c48ae0c9558cef734abf5cb34d6244378cddbf18e849d96
26aa394eea5630e07c48ae0c9558cef780d41e5e16056765bc8461851072c9d7
26aa394eea5630e07c48ae0c9558cef7ff553b5a9862a516939d82b3d3d8661a
c2261276cc9d1f8598ea4b6a74b15c2f57c875e4cff74148e4628f264b974c80
thread 'tests::check_whitelist' panicked at 'assertion failed: false', bin/node/runtime/src/lib.rs:2222:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
Seems we are missing:

```
// Treasury Account
				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da95ecffd7b6c0f78751baa9d281e0bfa3a6d6f646c70792f74727372790000000000000000000000000000000000000000").to_vec().into(),
```

Might need to think of something clever to be able to get that key in the list, since it is not a traditional storage item.

Reply:
for now, please append that one extra key to the generated list, and lets create a TODO to fix it

Reply:
done!